pinochle.go

```diff
@@ -76,7 +76,7 @@ const (
 var Faces [6]Face
 var Suits [4]Suit
 
-func init() {
+func Init() {
```

Comment:
init() is a special function in Go - I don't believe Init() will be treated the same.

https://golang.org/doc/effective_go.html#init